examples/plot_second_level_design_matrix.py

```diff
@@ -21,18 +21,18 @@
 # Create a simple experimental paradigm
 # --------------------------------------
 # Experimental paradigm has two conditions and 20 subjects
-column_names = ['map_name', 'subject_id', 'map_path']
+column_names = ['map_name', 'subject_label', 'map_path']
```

Comment:
Yes, to follow more closely the BIDS specification. ids refer to the whole string in the filenames like "sub-01" while label would refer only to "01". So here we are suppose to pass the subject_label and not the subject_id.